src/maps/maps-utils.ts

```diff
@@ -6,7 +6,7 @@ import { v4 as uuidv4 } from "uuid";
 export function processIMAGEMaps(document: Document, port: browser.Runtime.Port){
     Array.from(document.getElementsByTagName("iframe")).forEach(map => {
         if (!map.hasAttribute("tabindex") && !map.closest("a")) {
-            if (map.hasAttribute("src") && map.src.includes("google.com/maps")) {
+            if (map.hasAttribute("src") && (map.src.includes("google.com/maps") || map.src.includes("maps.google.ca/maps"))) {
```

Comment:
I think we could possibly come up a more general regex.
Is it ok if I put "map.src.includes("google") && map.src.includes("maps")"

Reply:
Generalizing definitely a good idea, but I suspect we'll run into trouble with something that general, since even a non-google URL could contain both those strings. Since I see that it can also be, for example `https://www.google.de/maps`, I'd say it would be safer to make a tighter regex, e.g.:
`[OPTIONAL maps.]google.[COUNTRYCODE,com]/maps` but to simplify, just check for any 2-letter combination instead of trying to do a canonical list of all possible COUNTRYCODEs.